anndata/tests/helpers.py

```diff
@@ -299,11 +300,6 @@ def are_equal_dataframe(a, b, exact=False, elem_name=None):
         b,
         check_index_type=exact,
         check_exact=exact,
-        # If False: check encoded values, but not codes
-        check_categorical=exact,
-        # Should different orderings be allowed?
-        # check_like=exact,
-        check_names=False,
```

Comment:
@ivirshup can you check if my merge is correct? I simplified this code simulataneously to you changing it.

Reply:
Looks fine to me!